pkg/alert/cfssldeployment.go

```diff
@@ -28,8 +28,8 @@ import (
 	"github.com/blackducksoftware/horizon/pkg/components"
 )
 
-// cfsslDeployment creates a new deployment for cfssl
-func (a *SpecConfig) cfsslDeployment() (*components.Deployment, error) {
+// getCfsslDeployment returns a new Deployment for a Cffsl
+func (a *SpecConfig) getCfsslDeployment() (*components.Deployment, error) {
```

Comment:
Since alert wants to use the same settings as hub, is the cfssl config different from the one in blackduck at all?  Can we make it common (maybe export the creation function from blackduck)?

Reply:
It seems like they are very similar. I think that we could create a helper function in pkg/utils that they both could use. Do you think it might be better if we leave all the Alert resources in one pkg so the Alert team doesn't have to modify other parts of the Synopsys-Operator repo if they want changes?

Reply:
I just talked to @msenmurugan . I am going to make a cfssl.go file in pkg/apps. Since this requires changes to Blackduck as well, I think it should be done in a separate PR from this one.